.github/workflows/build-preview.yml

```diff
@@ -0,0 +1,42 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. 
+on:
+  pull_request:
+    branches: [ master ]
```

Comment:
this will only fire on master, do we want to temporarily add the `docshubtwo` branch for testing the in-dev work for that project?

Reply:
Yep, give me time. This is WIP while I figure out how to publish preview sites, then we can adapt to the other branch